### PR TITLE
Adds assassinate subtypes that allow revival or cloning and dilutes assassinate objectives with them

### PR DESCRIPTION
**Pull request by SomeguyManperson**
opening in a new pr see #11132 for "discussion" since that one is hella old

# General Documentation

### Intent of your Pull Request

adds new objectives so you don't have to babysit or permadelete a corpse every time you get a kill objective just to not chance MAYBE POSSIBLY that someone found the body you hid, revived it without your knowledge, and gave the person a new identity or hiding place or whatever without you knowing
these allow greentext if the person has been cloned at least once, or if they have been dead at least once

"but theos this is hugbox"
shut the fuck up
"but theos this doesnt make sense"
see above
if we want people to take risks we need to make optimal play not the only optimal way to play the game

"but theos people might just make a deal with someone to quickly accomplish the objective"
yes I know our playerbase is incredibly soft and boring when it comes to making something interesting for themselves but that's a them problem not a me problem
also they can already do this by killing someone just before the round ends then reviving them afterwards
"but theos this isn't the best way to do this you should do x instead"
cool we are an open source codebase go ahead and start learning
### Why is this change good for the game?
it's not
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
two new objectives for solo antags: assassinate but cloning won't fail it and assassinate but any revival won't fail it. like maroon where there's more than one way to go about it.

# Changelog
:cl:  
rscadd: two new objectives that may take the place of assassinate objectives similarly to maroon, which function like assassinate objectives except they can succeed if the target has been cloned for one or has died at all for the other
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
but theos this isn't the best way to do this you should do x instead

**Comment by Cartlord** [comment]
Why would the Syndicate want this tho

**Comment by SomeguyManperson** [comment]
image

**Comment by yacabo** [comment]
Remember lads, you can ahelp for more/harder objectives

**Comment by Hopekz** [comment]
> Why would the Syndicate want this tho

Idk who cares other codebases like Eris have these types of objectives and it seems to somewhat work. let some lore people figure that part out.

That being said people still usually dispose of the bodies in those codebases even if it is optional so it probably won't do what @SomeguyManperson wants but who really knows this might be the first step to a solution that works (:

**Comment by SomeguyManperson** [comment]
> That being said people still usually dispose of the bodies in those codebases even if it is optional so it probably won't do what @SomeguyManperson wants but who really knows this might be the first step to a solution that works (:

I'd rather antags be hiding bodies for literally any reason except "the game is designed in such a way that removing this person from the round completely permanently is the only way  to ensure my victory"

**Comment by Moggram** [comment]
>
>
>>"but theos this doesnt make sense"
>>let some lore people figure that part out.

Sometimes its about sending a message. "We know who you are and can kill you at anytime."

**Comment by maxion12345** [comment]
so wait, defibs dont count but cloning does?

**Comment by adamsong** [comment]
> so wait, defibs dont count but cloning does?

Depends on t he objective, one of the new ones allows for revival defibbing, the other requires they get cloned

**Comment by SomeguyManperson** [comment]
> > > "but theos this doesnt make sense"
> > > let some lore people figure that part out.
> 
> Sometimes its about sending a message. "We know who you are and can kill you at anytime."

this apparently is a completely alien concept to some people


> so wait, defibs dont count but cloning does?

for kill once the objective completes if they are killed at all, for kill body the objective completes if they end the round dead or in a non-original body

**Comment by Cartlord** [comment]
The thing is, in the case of cloning, since we have Clone Memory Disorder, no message really gets sent - the deceased has no way of knowing the Syndicate is even trying to send a message, for all they know they might've just died from a workplace accident

**Comment by Moggram** [comment]
> 
> 
> The thing is, in the case of cloning, since we have Clone Memory Disorder, no message really gets sent - the deceased has no way of knowing the Syndicate is even trying to send a message, for all they know they might've just died from a workplace accident

"We are going to need you to play ball on [[redacted]](https://wiki.yogstation.net/wiki/Guide_to_xenobiology) and [[redacted]](https://1d4chan.org/wiki/Dark_Mechanicus) Just look at what happen to your friend."

**Comment by Hopekz** [comment]
> The thing is, in the case of cloning, since we have Clone Memory Disorder, no message really gets sent - the deceased has no way of knowing the Syndicate is even trying to send a message, for all they know they might've just died from a workplace accident

Again who cares some lore people will figure it out.

Who is to say that this message is even intended for the person getting killed and cloned they might just be someone important to the real target that is getting the message.
